pyppeteer/coverage.py

```diff
@@ -134,31 +134,36 @@ def __init__(self, client: CDPSession) -> None:
         self._eventListeners: List = list()
         self._resetOnNavigation = False
 
-    async def start(self, options: Dict = None, **kwargs: Any) -> None:
+    async def start(
+            self,
+            resetOnNavigation: bool = True,
+            reportAnonymousScripts: bool = False,
+    ) -> None:
         """Start coverage measurement."""
-        options = merge_dict(options, kwargs)
         if self._enabled:
             raise PageError('JSCoverage is always enabled.')
-        self._resetOnNavigation = (True if 'resetOnNavigation' not in options
-                                   else bool(options['resetOnNavigation']))
-        self._reportAnonymousScript = bool(options.get('reportAnonymousScript'))  # noqa: E501
+        self._resetOnNavigation = resetOnNavigation
+        self._reportAnonymousScript = reportAnonymousScripts
         self._enabled = True
         self._scriptURLs.clear()
         self._scriptSources.clear()
         self._eventListeners = [
             helper.addEventListener(
-                self._client, 'Debugger.scriptParsed',
-                lambda e: self._client._loop.create_task(
-                    self._onScriptParsed(e))),
+                self._client,
+                'Debugger.scriptParsed',
+                lambda e: self._client._loop.create_task(self._onScriptParsed(e))),
             helper.addEventListener(
-                self._client, 'Runtime.executionContextsCleared',
+                self._client,
+                'Runtime.executionContextsCleared',
                 self._onExecutionContextsCleared),
         ]
-        await self._client.send('Profiler.enable')
-        await self._client.send('Profiler.startPreciseCoverage',
-                                {'callCount': False, 'detailed': True})
-        await self._client.send('Debugger.enable')
-        await self._client.send('Debugger.setSkipAllPauses', {'skip': True})
+        await asyncio.gather(
+            self._client.send('Profiler.enable'),
+            self._client.send('Profiler.startPreciseCoverage',
+                              {'callCount': False, 'detailed': True}),
+            self._client.send('Debugger.enable'),
+            self._client.send('Debugger.setSkipAllPauses', {'skip': True}),
```

Comment:
Do you know if the order of this matters? It seems like the answer would be no, but if any of the items require order it's probably `Profiler.enable` and `Profiler.startPreciseCoverage`. Basically only the return order of the awaitables is guarenteed, not the sheduling order as I understand it (and I could definitely be wrong).

Reply:
Nevermind. It looks like the CPython implementation guarentee's order (although the spec does not techincally), I think it's pretty safe to rely on.

Reply:
yes the scheduling is ordered though execution of course will not be ordered but in this case I don't think the order matters. I've been just following js code and `Promise.all` seems to behave just like `asyncio.gather` so I think we're good to go here.